### DISCUSSION TITLE
solar production, grid import/export and home consumption

### DISCUSSION THREAD

**Discussion by purcell-lab** [alandtse/tesla#247] [Q&A] [answered]
Excited to see the solar & power options added.

Can you advise how I see those sensors in HA through this integration?

I can't see anything yet.

**Comment by denisjoshua** [comment]
Sorry if I ask here... the "INCLUDE ENERGY SITES" in the Tesla Integration is maybe the Tesla PowerWall2 integrations ? 
Thanks a lot 
Denis

**Comment by purcell-lab** [comment, marked as answer]
To close is Q&A with the release of v3.0.0 this integration now provides full sensors for my home energy gateway (powerwall 2), as well as my Tesla vehicle.

![Screenshot_20221023-203720](https://user-images.githubusercontent.com/79175134/197387408-22a1297e-6f11-4cdd-8d16-a43048c94522.png)

![Screenshot_20221023-203455](https://user-images.githubusercontent.com/79175134/197387338-8cd5e477-4d43-4ba3-b405-2c3c289d8b76.png)

![Screenshot_20221023-203509](https://user-images.githubusercontent.com/79175134/197387349-09bdf4e1-965b-4cee-be29-4f1a4dc8478f.png)

**Comment by denisjoshua** [comment]
Ok thanks a lot... but id I include "INCLUDE ENERGY SITES" in the Tesla Integration I take an error: 
```
Logger: custom_components.tesla_custom
Source: custom_components/tesla_custom/__init__.py:253
Integration: Tesla Custom Integration ([documentation](https://github.com/alandtse/tesla/wiki), [issues](https://github.com/alandtse/tesla/issues))
First occurred: 10:44:08 AM (4 occurrences)
Last logged: 10:58:28 AM

Unable to communicate with Tesla API: VEHICLE_UNAVAILABLE
```
How do I integrate it plese ? 
Is there a step-by-step manual ? 

Thanks again

**Comment by denisjoshua** [comment]
I still get same error... someone have some ideea please ? 
Thanks again 
Denis

**Reply by purcell-lab** [reply]
This discussion thread #247 had been closed.

You seem to be raising a new issue, if so could you please raise in the issue tab with some details of your system configuration and what you are expecting to see. Maybe some details logs.

Do you have a powewall and vehicle associated with your Tesla account?

**Reply by denisjoshua** [reply]
Thanks a lot for your answer... I'll make a new post then :-) 
Denis
